docs/api.md

```diff
@@ -501,13 +501,24 @@ puppeteer.launch().then(async browser => {
 });
 ```
 
-The Page class emits various events (described below) which can be handled using any of Node's native [`EventEmitter`](https://nodejs.org/api/events.html#events_class_eventemitter) methods, such as `on` or `once`.
+The Page class emits various events (described below) which can be handled using any of Node's native [`EventEmitter`](https://nodejs.org/api/events.html#events_class_eventemitter) methods, such as `on`, `once` or `removeListener`.
 
 This example logs a message for a single page `load` event:
 ```js
 page.once('load', () => console.log('Page loaded!'));
 ```
 
+To unsubscribe from events use the `removeListener` method:
+
+```js
+function logRequest() {
```

Comment:
Include the `request` param. Sorry to nit pic.

Reply:
No worries, would rather be more thorough